File: mycollab-services/src/main/java/com/mycollab/common/CountryValueFactory.java
package com.mycollab.common;

import java.util.Locale;

/**
 * @author MyCollab Ltd.
 * @since 1.0
 */
public class CountryValueFactory {
    private static String[] isoCountries;

    static {
        isoCountries = Locale.getISOCountries();
    }

    public static String[] getCountryList() {
        return isoCountries;
    }
}

File: mycollab-services/src/main/java/com/mycollab/common/CountryValueFactory.kt
package com.mycollab.common

import java.util.Locale

/**
 * @author MyCollab Ltd.
 * @since 1.0
 */
object CountryValueFactory {
    var countryList: Array<String>? = null
        private set

    init {
        countryList = Locale.getISOCountries()
    }
}
